src/AddOn.Episerver.Settings/Core/SettingsService.cs

```diff
@@ -396,7 +396,7 @@ private IEnumerable<IContent> LoadGlobalSettings()
                     this.log.Error($"[Settings] {ex.Message}", exception: ex);
                 }
 
-                if (item != null)
+                if (item != null && item.GetOriginalType() != typeof(SettingsBase))
```

Comment:
I don't understand this addition - why remove the content item from the global settings if it's using the base type?

Reply:
It is the way to detect if the setting has been created with the fallback to base because the settings class no longer exists. This is the way to allow deletion of such items so that the content type can be removed by the CMS.

Reply:
Ok, perhaps add a comment with this description for future reference? Other from that, it's gtg!